scripts/build_utils.sh

```diff
@@ -859,6 +865,12 @@ ceph_build_args_from_flavor() {
         echo "unknown FLAVOR: ${FLAVOR}" >&2
         exit 1
     esac
+
+    if [ -z "$WITH_JAEGER" ]; then
+      CEPH_EXTRA_RPMBUILD_ARGS="--with jaeger"
+      CEPH_EXTRA_CMAKE_ARGS+=" -DWITH_JAEGER=ON"
```

Comment:
i think this line is not necessary anymore. as you've passed this message down to `debian/rules` by setting `profiles` already.

the crimson flavor is not updated yet, that's why we are still using `CEPH_EXTRA_CMAKE_ARGS` to build the crimson flavor.

Reply:
makes sense, removed